frontend/src/components/GoesData/ComparisonModal.tsx

```
@@ -74,102 +67,103 @@ export default function ComparisonModal({
-                  <span className="font-medium text-gray-900 dark:text-white">{left.satellite}</span> · {left.band} · {new Date(left.capture_time).toLocaleString()}
-                </span>
-                <span>
-                  <span className="font-medium text-gray-900 dark:text-white">{right.satellite}</span> · {right.band} · {new Date(right.capture_time).toLocaleString()}
-                </span>
+            <div
+              ref={sliderRef}
+              className="relative bg-black rounded-lg overflow-hidden cursor-col-resize select-none"
+              style={{ height: '60vh' }}
+              onClick={handleSliderMove}
+            >
+              {/* Right image (full) */}
```

Biome (2.3.14)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)